### PR TITLE
remove logging of app environment without masking

### PR DESCRIPTION
**Pull request by ddonchev**
#### Description: 
The CF platform has no notion of sensitive environment variables. Therefore, without additional tracking there is no way to distinguish and mask the sensitive ones in log.

Until such a mechanism is implemented, all such tracing is removed.

### CODE REVIEW

File: com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/steps/BuildCloudDeployModelStep.java [hunk 1]
```diff
@@ -79,8 +79,10 @@ protected StepPhase executeStep(ProcessContext context) {
         // Build a list of applications for deployment and save them in the context:
         List<Module> modulesCalculatedForDeployment = calculateModulesForDeployment(context, deploymentDescriptor, mtaArchiveModules,
                                                                                     deployedModuleNames, mtaModules);
-
-        getStepLogger().debug(Messages.MODULES_TO_DEPLOY, secureSerializer.toJson(modulesCalculatedForDeployment));
+        List<String> moduleJsons = modulesCalculatedForDeployment.stream()
+                                                                 .map(module -> secureSerializer.toJson(module))
```
Comment (IvanBorislavovDimitrov) [inline com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/steps/BuildCloudDeployModelStep.java:83]: Method reference could be used: secureSerializer::toJson

File: com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/util/EnvironmentApplicationAttributeUpdater.java [hunk 2]
```diff
@@ -28,10 +28,8 @@ protected void updateAttribute(CloudApplication existingApplication, CloudApplic
     }
 
     private Map<String, String> applyUpdateStrategy(Map<String, String> existingEnv, Map<String, String> env) {
-        getLogger().debug(Messages.EXISTING_ENV_0, JsonUtil.toJson(existingEnv, true));
-        getLogger().debug(Messages.APPLYING_UPDATE_STRATEGY_0_TO_ENV_1, updateStrategy, JsonUtil.toJson(env, true));
+        getLogger().debug(Messages.APPLYING_UPDATE_STRATEGY_0_TO_ENV, updateStrategy );
```
Comment (IvanBorislavovDimitrov) [inline com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/util/EnvironmentApplicationAttributeUpdater.java:31]: Could you format this?

File: com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/util/EnvironmentApplicationAttributeUpdater.java [hunk 3]
```diff
@@ -28,10 +28,8 @@ protected void updateAttribute(CloudApplication existingApplication, CloudApplic
...
         Map<String, String> result = getElementUpdater().updateMap(existingEnv, env);
-        getLogger().debug(Messages.RESULT_0, JsonUtil.toJson(result, true));
         return result;
```
Comment (IvanBorislavovDimitrov) [inline com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/util/EnvironmentApplicationAttributeUpdater.java:33]: The result can be returned directly.

File: com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/steps/BuildCloudDeployModelStep.java (hunk 1 above)
Comment (ddonchev) [reply com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/steps/BuildCloudDeployModelStep.java]: 10x!

File: com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/util/EnvironmentApplicationAttributeUpdater.java (hunk 3 above)
Comment (ddonchev) [reply com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/util/EnvironmentApplicationAttributeUpdater.java]: yep :)

**Pull request merged**